docs/userguide/009_JUnit_Support.adoc

```diff
@@ -156,6 +156,8 @@ public class ArchitectureTest {
 }
 ----
 
+Note that JUnit's `@Disabled` annotation is not supported. Instead, `@ArchIgnore` should be used.
```

Comment:
Thanks for the fast reaction! 👍

Just one thought: As the section is about JUnit **4 & 5**, I wonder whether we should more specfically refer to
> Junit 5's `@Disabled`

here? On the other hand, this probably shouldn't confuse JUnit 4 users, so I don't mind keeping the note simple.

Reply:
Extending the docs is a good idea :+1: Since this section is a bout JUnit 4 and 5, what about writing sth. like "For JUnit 5 note that ArchUnit does not support JUnit Jupiter annotations like `@Disabled`, but provides its own API" or something like that? :thinking: I think it would be good to somehow make it clear that this is a pure JUnit 5 comment. But maybe that same sentence with a prefix "Note for JUnit 5 users:" or similar would be good enough?